src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -109,22 +109,20 @@ public class AccordJournal implements accord.api.Journal, RangeSearcher.Supplier
     @VisibleForTesting
     protected final AccordJournalTable<JournalKey, Object> journalTable;
     private final Params params;
-    private final AccordAgent agent;
     Node node;
 
     enum Status { INITIALIZED, STARTING, REPLAY, STARTED, TERMINATING, TERMINATED }
     private volatile Status status = Status.INITIALIZED;
 
-    public AccordJournal(Params params, AccordAgent agent)
```

Comment:
took awhile but it did turn out the agent is dead code; tons of code to cleanup to remove that link